static/components/AppAround.js

```diff
@@ -108,8 +108,12 @@
 	        });
 	      });
 	      navigator.geolocation.getCurrentPosition(function (position) {
+	        var userPosition = [position.coords.longitude, position.coords.latitude];
+	        moveTo(userPosition);
+	        currentMarker(userPosition);
+	        console.log(userPosition);
 	        _this2.setState({
```

Comment:
Instead of using `_this`, `_this2`, … you can simply use arrow functions:
```
fetch('/around/spots.json').then(response => {
  …
  map.on('moveend', move => {
  …
  navigator.geolocation.getCurrentPosition(position => {
```